packages/common/src/i18n/es.json

```
@@ -11,5 +15,67 @@
+    },
+    "nationalChildAbuseHotline": {
+      "name": "Línea Nacional de Abuso Infantil",
+      "description": "La línea directa nacional de abuso infantil “Childhelp” 1-800-4-A-CHILD (1-800-422-4453) está dedicada a la prevención del abuso infantil. Sirviendo en los Estados Unidos, territorios estadounidense no incorporados y Canadá. La línea directa cuenta con personal las 24 horas del día, los 7 días de la semana.  Los asesores son profesionales de crisis que gracias a intérpretes, pueden brindar asistencia en más de 170 idiomas. La línea directa ofrece apoyo en momentos de crisis, información general acerca de abuso infantil, y referencias a miles de emergencias, servicios sociales y recursos de apoyo. Todas las llamadas son confidenciales."
+    },
+    "nationalRunawaySafeline": {
+      "name": "Linea Nacional del Programa Fugitivo (“Runway”)",
+      "description": "Linea Nacional del Programa “Runaway” está aquí para apoyar a cualquier joven o adolescente que está pensando en huir o si ya lo ha hecho. Nuestros servicios son confidenciales y sin prejuicios. Para los padres y tutores, la Linea Nacional del Programa “Runaway”  puede ofrecerle ayuda y conectarlo con los recursos que su familia necesita. Ya sea que tenga problemas para hablar con su hijo o que su hijo este fugitivo, “Runaway” puede ayudarte. Te escucharemos y te ayudaremos a desarrollar un plan."
+    },
+    "glbtNationalHotline": {
+      "name": "Línea Directa Nacional de Gays, Lesbianas, Bisexuales y Transgénero (GLBT)",
+      "description": "La línea directa nacional de gays, lesbianas, bisexuales y transgénero (GLBT) brinda apoyo telefónico, chat privado en línea y ayuda por correo electrónico. Te brindamos información objetiva y recursos locales en ciudades y pueblos de los Estados Unidos. Todos nuestros servicios son gratuitos y confidenciales. ¡Hablamos con personas que llaman de cualquier edad acerca de cualquier tema que surja como problemas en su relación, acoso en la escuela, problemas en su lugar de trabajo, ansiedad por el VIH / SIDA e información sobre sexo seguro, y mucho más! Nosotros tenemos un grupo diverso de voluntarios del grupo GLBT, operamos dos líneas directas nacionales, la Línea directa nacional de GLBT (1-888-843-4564) y la Línea de conversación nacional juvenil de GLBT (1-800-246-7743). Tambien usamos una sala de chat privada y voluntaria, que ayuda tanto a jóvenes y adultos. Nuestro correo electrónico es  help@GLBThotline.org"
```

```suggestion
      "description": "La línea directa nacional de gays, lesbianas, bisexuales y transgénero (GLBT) brinda apoyo telefónico, chat privado en línea y ayuda por correo electrónico. Te brindamos información objetiva y recursos locales en ciudades y pueblos de los Estados Unidos. Todos nuestros servicios son gratuitos y confidenciales. ¡Hablamos con personas que llaman de cualquier edad acerca de cualquier tema que surja como problemas en su relación, acoso en la escuela, problemas en su lugar de trabajo, ansiedad por el VIH / SIDA e información sobre sexo seguro, y mucho más! Nosotros tenemos un grupo diverso de voluntarios del grupo GLBT, operamos dos líneas directas nacionales, la Línea directa nacional de GLBT (1-888-843-4564) y la Línea de conversación nacional juvenil de GLBT (1-800-246-7743). Tambien usamos una sala de chat privada y voluntaria, que ayuda tanto a jóvenes y adultos. Nuestro correo electrónico es help@GLBThotline.org"
```
Nit: Removing extra space.